website/docs/hardware/charging.md

```diff
@@ -5,8 +5,8 @@ title: Batteries and Charging
 
 ## Batteries
 
-Your Amiga use two batteries inside forks A and B, but two more can be added on forks C and D
-for more energy demanding applications. The specifications for each battery are:
+Your Amiga use two batteries, typically inside forks A and B, but it can be changed. Verify at
+your Amiga where the battery mount plates are. The specifications for each battery are:
```

Comment:
More readable / maintainable to end lines on punctuation marks and start new sentences on new lines

```suggestion
Your Amiga use two batteries, typically inside forks A and B, but it can be changed.
Verify on your Amiga where the battery mount plates are.
The specifications for each battery are:
```

Reply:
Ditto throughout this PR, but feel free to ignore so you can get this merged